cmd/env/get.go

```diff
@@ -95,6 +95,42 @@ func NewGetAllEnvironmentsCmd() *cobra.Command {
 
 var ErrUnmarshalling = errors.New("failed to unmarshal environment(s)")
 
+// Converts the `environment` object to [][]string which is used during printing environments as table
+func convertObjToArray(env struct{ environment }) [][]string {
+	var data [][]string
+
+	for _, p := range env.Attributes.Projects {
+		pr := strconv.Itoa(p.PullRequestNumber)
+		if pr == "0" {
+			pr = ""
+		}
+
+		data = append(data, []string{
+			env.Attributes.Name,
+			env.ID,
+			fmt.Sprintf("%t", env.Attributes.Ready),
+			p.RepoName,
+			pr,
+			env.Attributes.URL,
+		})
+	}
+
+	return data
+}
+
+// Render the environment data in tabular form
+func renderTable(data [][]string) {
+	table := tablewriter.NewWriter(os.Stdout)
+	table.SetHeader([]string{"App Name", "UUID", "Ready", "Repo", "PR#", "URL"})
+	table.SetAutoMergeCellsByColumnIndex([]int{0, 1, 5})
```

Comment:
Did you remove the borders to allow for separation of projects within one environment row?
I think it'd be nice to keep the minimal borders, if possible, so as to have less visual ASCII noise...

Reply:
Yeah, used the example for `SetAutoMergeCellsByColumnIndex` which had some noise.
Cleared it up a bit. Let me know what you think about this.

Reply:
I like it! It's minimal and simple. We can always tweak those, of course. This, surprisingly, has a huge effect on the look of the output!